tests/std/tests/GH_001010_filesystem_error_encoding/test.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <filesystem>

```

Comment:
Technically need to include `<exception>` for `std::exception` below:
```suggestion
#include <assert.h>
#include <exception>
#include <filesystem>
```